src/server/plugins/engine/views/index.html

```diff
@@ -14,6 +17,10 @@
         {{ govukErrorSummary(errors) }}
       {% endif %}
 
+      {% if isStartPage %}
+        {{ warnMissingNotificationEmail(designerUrl, slug) }}
+      {% endif %}
```

Comment:
What if someone clicks "Preview" (in **forms-designer**) on a question page?

They'll miss the warning. This might be a requirement, just making sure

Reply:
Just confirming this is the desired behavior from the story. First and summary page only.